README.md

```diff
@@ -5,7 +5,7 @@ The **Platform.sh CLI** is the official command-line interface for [Platform.sh]
 ## Requirements
 
 * Operating system: Linux, OS X, or Windows 10
-* PHP 5.5.9 or higher, with cURL support
+* PHP 5.5.9 or higher, with `cURL` `json` `phar` `mbstring` support
```

Comment:
```suggestion
* PHP 5.5.9 or higher, with the following extensions: `curl`, `json`, `mbstring`, `pcre`, and `phar`. The installation command will check for these.
```

with some notes

- `phar` is only required for the recommended installation method, it's not technically a hard requirement (e.g. you could install the CLI via Composer).
- I believe `curl` support is not really a hard requirement, the CLI uses Guzzle more or less throughout which can fall back to the built-in `fopen` transport. But I haven't tried the CLI without curl. I think it makes certain things more difficult (like configuring TLS root certs?).
- one of `mbstring` or `iconv` is required, apparently for Symfony Console. I think `mbstring` is newer and more widely available/preinstalled.
- thankfully `json` will always be included in PHP 8.